source/LoadPanel.cpp

```diff
@@ -167,40 +169,77 @@ void LoadPanel::Draw()
 	if(loadedInfo.IsLoaded())
 		info.SetCondition("pilot loaded");
 
+	const Interface *loadPanel = GameData::Interfaces().Get("load menu");
+
 	GameData::Interfaces().Get("menu background")->Draw(info, this);
-	GameData::Interfaces().Get("load menu")->Draw(info, this);
+	loadPanel->Draw(info, this);
 	GameData::Interfaces().Get("menu player info")->Draw(info, this);
 
-	// The list has space for 14 entries. Alpha should be 100% for Y = -157 to
-	// 103, and fade to 0 at 10 pixels beyond that.
-	Point point(-470., -157. - sideScroll);
-	const double centerY = font.Height() / 2;
-	for(const auto &it : files)
+	// Draw the list of pilots.
 	{
-		Rectangle zone(point + Point(110., centerY), Point(230., 20.));
-		bool isHighlighted = (it.first == selectedPilot || (hasHover && zone.Contains(hoverPoint)));
-
-		double alpha = min(1., max(0., min(.1 * (113. - point.Y()), .1 * (point.Y() - -167.))));
-		if(it.first == selectedPilot)
-			FillShader::Fill(zone.Center(), zone.Dimensions(), Color(.1 * alpha, 0.));
-		font.Draw({it.first, {220, Truncate::BACK}}, point, Color((isHighlighted ? .7 : .5) * alpha, 0.));
-		point += Point(0., 20.);
+		// The list has space for 14 entries. Alpha should be 100% for Y = -157 to
+		// 103, and fade to 0 at 10 pixels beyond that.
```

Comment:
The fading causes the last entry to be a bit darker than the rest, which looks a bit weird I think.

Reply:
> Fix fading, maybe?

nope

Reply:
Looks fine to me. Scrolled all the way to the top, there's no fading on the lowest line. If I navigate with the keyboard, nothing ever fades. I only get fading if I scroll with the mouse, and when doing that it looks indistinguishable from the current game to me.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/19e981d2-c46b-45d1-9226-74455a50b3ee)